compiler-core/src/parse/snapshots/gleam_core__parse__tests__deprecation_attribute_on_all_type_variants.snap

```diff
@@ -0,0 +1,17 @@
+---
+source: compiler-core/src/parse/tests.rs
+assertion_line: 1512
+expression: "\ntype Wibble {\n    @deprecated(\"1\")\n    Wibble1\n    @deprecated(\"2\")\n    Wibble2\n}\n"
+---
+error: Syntax error
+  ┌─ /src/parse/error.gleam:2:1
+  │
+2 │ type Wibble {
+  │ ^^^^^^^^^^^ Can't deprecate all variants of a type.
+
+Consider deprecating the type as a whole:
+@deprecated("message")
+type Wibble {
+   Wobble1
+   Wobble2
+}
```

Comment:
Really nice suggestion! Usually in the error reporting the convention is that multiline code snippets are indented by 4 spaces and preceded by an empty line, so this should be:
```suggestion
Consider deprecating the type as a whole:

    @deprecated("message")
    type Wibble {
      Wobble1
      Wobble2
    }
```